src/featureSelectors.test.ts

```diff
@@ -29,13 +29,40 @@ describe("featureSelectors", () => {
         name: "users",
         entity: "users",
       });
+
       const output = getDenormalizedData.resultFunc(featureSlices, true);
+
       expect(output).toEqual({
         users: [
           { id: 1, name: "Garrett" },
           { id: 2, name: "Laura" },
         ],
       });
     });
+
+    it("should return undefined if the selected slice has a falsy status", () => {
```

Comment:
Good titles for tests should tell the user what they're testing without being too specific. Here's a title that I think would be appropriate: "should return falsy value when the status of the slice is not yet defined"